packages/trader/src/Stores/Modules/SmartChart/smart-chart-store.js

```diff
@@ -17,10 +17,11 @@ import BaseStore              from '../../base-store';
 export default class SmartChartStore extends BaseStore {
     @observable chart_type;
     @observable granularity;
-    @observable barriers = observable.object({});
-    @observable markers  = observable.object({});
-    barriers_empty_array = [];
-    markers_empty_array = [];
+    @observable barriers  = observable.object({});
+    @observable markers   = observable.object({});
+    barriers_empty_array  = [];
+    markers_empty_array   = [];
+    subscription_requests = [];
```

Comment:
@aminroosta does this look fine to you?

Reply:
@negarn , `smart-chart-store` will be killed in #276  so these could be moved to `trade-store` and connected from there instead.

Reply:
In any case the subscription user is now required to keep the `subscriber` objects to call `subscriber.unsubscribe()` on them after they're done with the subscription.

Me and Amin talked about this, and he was confident that these changes can be used in his PR.

Reply:
@negar-binary @aminmarashi-binary bascially we will override trade-store, contract-trade-store & contract-replay-store with our branch ... we'll just have a look at PR to get an idea of what we need to change.

@aminmarashi-binary lets sit together next week and merge your changes into the multiple-contract PR.

Reply:
:crossed_fingers: